myft-hint/main.scss

```diff
@@ -0,0 +1,73 @@
+
+@include nUiCritical('myftHint') {
+		.myft-hint {
+			display: none;
+		}
+};
+
+	@keyframes bounce {
+		from {
+			right: 90px;
+		}
+
+		to {
+			right: 100px;
+		}
+	}
+
+	.myft-hint {
+		animation-duration: 1s;
+		animation-name: bounce;
+		animation-iteration-count: infinite;
+		animation-direction: alternate;
+	}
+
+	.myft-hint__container {
+		background-color: getColor('white');
+		border: 1px solid getColor('pink-tint4');
+		white-space: nowrap;
+		position: relative;
+		border-radius: 5px;
+		box-shadow: -1px 1px getColor('pink-tint4');
+		&::before {
+			width: 0;
+			height: 0;
+			border-top: 12px solid transparent;
+			border-bottom: 12px solid transparent;
+			border-left: 12px solid getColor('pink-tint4');
+			content: '';
+			position: absolute;
+			top: 33%;
+			left: 100%;
+		}
+		&::after {
+			width: 0;
+			height: 0;
+			border-top: 10px solid transparent;
+			border-bottom: 10px solid transparent;
+			border-left: 10px solid getColor('white');
+			content: '';
+			position: absolute;
+			top: 35%;
+			left: 100%;
+		}
+	}
+
+	.myft-hint__content {
+		padding: 0;
+		margin: 8px 12px 8px 8px;
+		width: 100%;
+		font-size: 0.8em;
+		font-family: Georgia, serif;
```

Comment:
Could these font styles be replaced with some o-typography stuff? https://github.com/Financial-Times/o-typography